packages/graphql/lib/utils/types.ts

```diff
@@ -0,0 +1,2 @@
+export type Many<T> = T | readonly T[];
```

Comment:
```suggestion
export type Many<T> = T | ReadonlyArray<T>;
```

this should work. See https://github.com/nestjs/nest/pull/13655